Header header
Vector2 imgSize
Vector2 center
Vector2 bottomCenter
Vector2[] contour
float32 relativeArea